client/src/theme.css

```
@@ -1,151 +1,197 @@
 .theme-bg {
-  background: linear-gradient(135deg, #020617, #0f172a, #020617);
+  background-color: #050505;
   color: #f8fafc;
   min-height: 100vh;
   position: relative;
   overflow-x: hidden;
+  font-family: 'Inter', -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)